### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by delfipolito**
Search input on the docs side bar

### CODE REVIEW

File: lib/static/css/main.css
Comment (AquiGorka) [inline lib/static/css/main.css:952]: Suggestion to follow a mobile-first approach:

```css
/* first define the base styles */
.navSearchWrapper {
  display: none;
}
/* override when certain breakpoint is reached (1010px in this case) */
@media only screen and (min-width: 1010px) {
  .navSearchWrapper {
    display: block;
  }
 }
```
I see the codebase uses `max-width` `media-queries` so maybe mixing up is not the best thing, your call, I did want to suggest here because we follow a _mobile-first_ convention in the other repos.

File: lib/static/css/main.css
Comment (AquiGorka) [inline lib/static/css/main.css]: Indentation is a bit off here.

File: lib/static/css/main.css
Comment (delfipolito) [reply lib/static/css/main.css:952]: Yes, I think is best to follow a mobile-first approach, but I would do it in a separate refactor PR, modifying all media queries at once

**Review by AquiGorka — APPROVED** [review]

File: lib/static/css/main.css
Comment (AquiGorka) [inline lib/static/css/main.css:849]: I will always ask about `!important`, I believe you should avoid it at all costs and will try to figure out if the reason it is needed is because of css specificity instead of simply adding the important overlord because once you add one, the only way to override it is to use it again.
Having said all of that, why is it needed here? Can you make it work without it?

File: lib/static/css/main.css
Comment (AquiGorka) [inline lib/static/css/main.css:1753]: Can you try using more specificity instead of `!important` here?

**Review by AquiGorka — APPROVED** [review]
All good 😄 
I left a couple of comments regarding `!important`, I really suggest you try to avoid it and if you can, there is no need for another review; if you can't remove them, then yo are good to go too.

**Pull request merged**
